repose-aggregator/core/core-lib/pom.xml

```diff
@@ -137,6 +138,18 @@
 
         <dependency>
             <groupId>org.openrepose</groupId>
+            <artifactId>atom-feed-service-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
why both? the only thing i can think of is that there is a reference to something in api